__tests__/endpoints.test.js

```diff
@@ -92,6 +91,40 @@ describe('GET /api/articles/:article_id', () => {
       });
 });
 
+describe('GET /api/articles', () => {
+  test('Should return an articles array of all article objects with all properties except the body property', () => {
+    return request(app).get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body;
+        expect(Array.isArray(articles)).toBe(true)
+        expect(articles).toHaveLength(13)
+        articles.forEach(article => {
+          expect(article).toHaveProperty('author');
+          expect(article).toHaveProperty('title');
+          expect(article).toHaveProperty('article_id');
+          expect(article).toHaveProperty('topic');
+          expect(article).toHaveProperty('created_at');
+          expect(article).toHaveProperty('votes');
+          expect(article).toHaveProperty('article_img_url');
+          expect(article).toHaveProperty('comment_count')
+        });                
+      });
```

Comment:
This is great so far, you are asserting everything that should be within each article, however just as your description for this test mention, body should not be included so having an assertion to ensure this is the case can be added here as well.